### DISCUSSION TITLE
How to automatically open a reference at the center of the screen?

### DISCUSSION THREAD

**Discussion by shellwhale** [nvimdev/lspsaga.nvim#1474] [Q&A]
When using `Lspsaga finder` and pressing `o`, the cursor goes to the location of the reference but the line is not centered.
I usually press `zz`.

Is there a way to automate this?

**Comment by glepnir** [comment]
seems like a bug ?

**Reply by shellwhale** [reply]
Oh, so that's not the intended behavior.
